content/getting-started/install/linux.md

```diff
@@ -39,12 +39,6 @@ wget https://github.com/tinygo-org/tinygo/releases/download/v0.21.0/tinygo_0.21.
 sudo dpkg -i tinygo_0.21.0_arm.deb
 ```
 
-You will need to ensure that the path to the `tinygo` executable file is in your `PATH` variable.
-
-```shell
-export PATH=$PATH:/usr/local/tinygo/bin
-```
-
```

Comment:
@lomoalbert 
Is there any reason to remove this part?

Reply:
This PR can probably be closed. This seems to be superseded by d1d2c7483451b3bacf9c59558e116c218782ba4f on Mon Mar 7 04:59:24 2022 which replaced the old `/usr/local/tinygo/bin` with the correct `/usr/local/bin`.